source/_integrations/lawn_mower.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Lawn Mower
+description: Instructions on how to setup and use lawn mowers in Home Assistant.
+ha_release: 2023.10
+ha_domain: lawn_mower
+ha_quality_scale: internal
+ha_category:
+  - Lawn Mower
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+The `lawn_mower` integration enables the ability to control home lawn mowers within Home Assistant.
```

Comment:
I suggest taking a quick look at the [event documentation](https://www.home-assistant.io/integrations/event), as that provides the most recent way to approach this.

We need to make it really clear this is not something you can create yourself, but is an entity that is provided by other integrations.

Reply:
Ah yes, I'll take a look asap in the morning.

Reply:
Can you please review the top sentence as well, I think it is an overall better sentence but open to changing it back.